### ISSUE TITLE
runtests.py arguments not working

### ISSUE THREAD

**Issue by millerta** [lanl/LaGriT#248]
python runtests.py
works for default args, level 1 with exe in ../build/lagrit

but exits early when trying to use arguments:
```
python runtests.py -exe ../tmpdir/lagrit
usage: runtests.py [-h] [-v] [-q] [--locals] [-f] [-c] [-b] [-k TESTNAMEPATTERNS] [tests [tests ...]]
runtests.py: error: unrecognized arguments: -exe
```

According to lagrit-tests.log, it looks like args were set, so something else is causing early exit.
```
DEBUG:Called with args: Namespace(executable='../tmpdir/lagrit', levels=[1])
```

**Comment by millerta** [comment]
At the very least, the python script should work with given path. Currently no options work:

es19:LaGriT_V333:bsh% python test/runtests.py --help
usage: runtests.py [-h] [-l [LEVELS [LEVELS ...]]] [-exe EXECUTABLE]

LaGriT test suite

optional arguments:
  -h, --help            show this help message and exit
  -l [LEVELS [LEVELS ...]], --levels [LEVELS [LEVELS ...]]
                        designate level(s) of testing. Example: --levels 1 2
  -exe EXECUTABLE, --executable EXECUTABLE
                        path to LaGriT executable for testing
es19:LaGriT_V333:bsh% python test/runtests.py -exe "/project/eesdev/tam/clone/LaGriT_V333/build_noexodus/lagrit"
usage: runtests.py [-h] [-v] [-q] [--locals] [-f] [-c] [-b] [-k TESTNAMEPATTERNS]
                   [tests [tests ...]]
runtests.py: error: unrecognized arguments: -exe
es19:LaGriT_V333:bsh% python test/runtests.py --executable /project/eesdev/tam/clone/LaGriT_V333/build_noexodus/lagrit
usage: runtests.py [-h] [-v] [-q] [--locals] [-f] [-c] [-b] [-k TESTNAMEPATTERNS] [tests [tests ...]]
runtests.py: error: unrecognized arguments: --executable

**Comment by millerta** [comment]
Erica has fixed and improved the test suite. It is included in new release V3.3.3
